docs/tutorials/tutorial_observables.ipynb

```diff
@@ -0,0 +1,1572 @@
+{
```

Comment:
<p>I think this section connects the tutorial very nicely to the robinson model from the previous tutorials and conveys the economic intuition in a elegant manner :).Maybe we could use a different letter than delta though since this is usually associated with the discount factor? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/367/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Completely neglected that, I'll change it